### ISSUE TITLE
Voiceover is not announcing 'Responsible Computing and User Agreement' text as heading in the PDF in iOS.

### ISSUE THREAD

**Issue by msftedad** [mozilla/pdf.js#19168]
### Attach (recommended) or Link to PDF file

https://mozilla.github.io/pdf.js/web/viewer.html

### Web browser and its version

Safari Version 18.1.1

### Operating system and its version

iPhone 13 Version 18.1.1

### PDF.js version

1.4

### Is the bug present in the latest PDF.js version?

Yes

### Is a browser extension

Yes

### Steps to reproduce the problem

1. Open https://mozilla.github.io/pdf.js/web/viewer.html pdf (Having links in it (e.g.: Responsible Computing User Agreement 5-4-2022 (1) (1).pdf)).
2. Use swipe gestures to move to the 'Responsible Computing and User Agreement' text.
3. Verify using swipe gestures whether Voiceover is not announcing 'Responsible Computing and User Agreement' text as heading in the PDF or not.

### What is the expected behavior?

Voiceover should announce the 'Responsible Computing and User Agreement' text as heading in the PDF in iOS.

### What went wrong?

Voiceover is not announcing 'Responsible Computing and User Agreement' text as heading in the PDF in iOS.

https://github.com/user-attachments/assets/4ad1fd71-1449-47c8-aa24-d2ca49dc8ac4

**Comment by Snuffleupagus** [comment]
https://github.com/mozilla/pdf.js/issues/19166#issuecomment-2519774244 applies here as well.
